installation.md

```diff
@@ -0,0 +1,19 @@
+# Installation
+First, you will need to access the Home Assistant files, for example, using ssh.
+An useful tutorial is available here: [https://www.wundertech.net/how-to-ssh-into-home-assistant/](https://www.wundertech.net/how-to-ssh-into-home-assistant/)
+
+1. Access the terminal and paste the commands bellow:
+```
+cd ~/config/custom_components/
+download_url=$(wget -q -O - https://api.github.com/repos/darkxst/ihost_indicator/releases/latest | jq -r '.assets[] | select(.name | contains("tar.xz")) | .browser_download_url')
+wget "$download_url"
+filename="${download_url##*/}"
+tar -xf "$filename"
+```
```

Comment:
This is bit complex to drop on the end user, can you wrap it up in a shell script and put in root of repo.

This way the user could just copy and run a single command to install

```wget <repo>/install.sh | sh install.sh```

Maybe also just include the installation within the main README